cypress/e2e/inventory/search/browse-search-permissions.cy.js

```diff
@@ -0,0 +1,53 @@
+import TopMenu from '../../../support/fragments/topMenu';
+import TestTypes from '../../../support/dictionary/testTypes';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import devTeams from '../../../support/dictionary/devTeams';
+import Users from '../../../support/fragments/users/users';
+import permissions from '../../../support/dictionary/permissions';
+
+describe('permissions: inventory', () => {
+    let firstUser;
+    let secondUser;
+
+  before(() => {
+    cy.createTempUser([
+        permissions.uiInventoryViewInstances.gui,
+      ]).then(userProperties => {
+        firstUser = userProperties;
+      });
+    cy.createTempUser([
+        permissions.uiInventoryViewInstances.gui,
```

Comment:
I see in test case
"A user with "Inventory: All permissions" permission is logged in"

So I assume here should be inventoryAll isntead of uiInventoryViewInstances

Reply:
Done